sbin/load-celeborn-env.sh

```diff
@@ -69,3 +69,25 @@ if [ "$CELEBORN_PID_DIR" = "" ]; then
   export CELEBORN_PID_DIR="${CELEBORN_HOME}/pids"
 fi
 
+# jemalloc memory allocator is enabled by default, you may set DISABLE_JEMALLOC to true if it's not as you wish.
+DISABLE_JEMALLOC="false"
```

Comment:
I suppose this declaration should be removed? otherwise, it will overwrite the definition in `celeborn-env.sh`